File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    len_p = len(p)
    split_num_lst = []
    result = 0
    
    for i in range(len(t) - len_p + 1):
        split_num_lst.append(int(t[i:i+len_p]))
    
    for num in split_num_lst:
        if num <= int(p): result += 1
    
    return result

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    const p_len = p.length;
    const split_num_lst = [];
    let result = 0;
    
    for (let i=0; i<t.length - p_len + 1; i++) {
        split_num_lst.push(Number(t.slice(i, i+p_len)));
    }
    
    split_num_lst.forEach((num) => {
        if (num <= Number(p)) {
            result += 1;
        }
    });
    return result;
}